### ISSUE TITLE
The project should not be named yolov5！

### ISSUE THREAD

**Issue by amusi** [ultralytics/yolov5#2]
1. I think the project name should not be called yolov5
1. Please add a link to yolov4：https://github.com/AlexeyAB/darknet

**Comment by WDNMD0-0** [comment]
what you said is awesome,but you can you up,no can no bb.

**Comment by wangtianlong1994** [comment]
I suggest you write a yolov6 to beat yolov5

**Comment by Mechanic-Hwang** [comment]
Maybe yolov999999999 will be great
or yolov666666666,  Thumbs up, Thumbs up, Thumbs up

**Comment by ethanyanjiali** [comment]
I'm not sure if the author is mocking YOLO v4, but why not ultralytics-net, or ultralytics-yolo? it will become much easier for us to organize our model zoo.

btw I noticed the new Chinese translation "你只看一次" on the cover page, but that sounds a bit awkward in Chinese. Personally i like to call it "一眼万年", which means YOLO just need one glance while other two-stage detectors runs like 10k years.

just jk

**Comment by Mechanic-Hwang** [comment]
I also think the translation of '你只看一次' was not good, The translation is not necessary. YOLO is great.

**Comment by IgorSusmelj** [comment]
I also think yolov5 is misleading. A new version should require a publication with proper benchmarks against previous versions and significant changes/ improvements of the architecture (not just parameter tuning and minor changes). I don't see any of this.

Why not name it **yolov4-ultralytics**?

**Comment by huynhbach197** [comment]
Where is your paper, publication?

**Comment by JonnoFTW** [comment]
This repository simply implements the existing published yolov4 architecture in a new framework. There is no substantial novel contribution made here to the architecture that warrants a new name.

**Comment by glenn-jocher** [comment]
@amusi and all others regarding publication and naming:

Thank you for your feedback! Regarding publication, we would very much like to be able to publish a paper detailing the various modifications employed to achieve these results (among all 3 major components: architecture, loss function and training methodology), however we are extremely limited on resources, and thus must smartly select how best to deploy these in order to keep our business viable as a going concern, while also continuing our work of pushing the boundaries of what's possible in this and other fields. 

Importantly **these models are neither static nor complete** at this time. Our recent open-sourcing of this work is simply part of our ongoing research, and is not any sort of final product, and for this reason it is not accompanied by any publication. Our current goal is to continue internal R&D throughout the remainder of 2020, and hopefully open source and publish at least a short synopsis of this to Arxiv by the end of the year. 

We have published several peer reviewed [works](https://www.ultralytics.com/publications) prior as a company, and myself personally as well ([Google Scholar](https://scholar.google.com/citations?hl=en&user=swSrGtsAAAAJ)), mostly in particle physics and antineutrino detection. Our most notable publication was the world's first [Antineutrino Global Map](https://www.nature.com/articles/srep13945), published in Nature, Scientific Reports as part of ongoing work Ultralytics performed for the U.S. National Geospatial-Intelligence Agency.

It was our work reconstructing neutrino events which led us to develop an ancillary interest in AI, and in this field we are pursuing research that is both transparent and reproducible, which builds on years of hard work (by myself and many community contributors) porting and perfecting YOLOv3 to PyTorch from Darknet, which pjreddie amazingly began and @alexeyab so impressively continued and advanced. Most importantly in following the spirit of YOLO, we are targeting results that we hope may be **in reach of every individual** who would express an interest in the field, not merely available to those with unlimited resources at their disposal. Our smallest model, to cite an example, trains on COCO in only 3 days on one 2080Ti, and runs inference faster and more accurately than EfficientDet D0, which was trained on 32 TPUv3 cores by the Google Brain team. By extension we aim to comparably exceed D1, D2 etc. with the rest of the YOLOv5 family.

Regarding naming, we do take note of the comments surrounding the issue. YOLOv5 is an internal designation assigned to this work, which is now open-sourced. The exact name employed here is not a concern for us (we are open to alternatives!), we are instead focused on producing, improving, and delivering results to our clients, and to the open-source community by extension when our contract terms allow for it, which we push for often.

We appreciate all feedback, and we will try to be as responsive as we can going forward!

**Comment by glenn-jocher** [comment]
Ah, yes, many people were wondering about the Chinese translation. The story behind this is that we noticed that more than half of all visitors came from China, so we thought it would it would be nice to display a Chinese translation as a welcome for them.

<img width="439" alt="Screen Shot 2020-06-10 at 11 27 24 PM" src="https://user-images.githubusercontent.com/26833433/84352317-f9462780-ab71-11ea-8faa-d04de388ca76.png">

We did this by using Google Translate.
<img width="621" alt="Screen Shot 2020-06-10 at 11 32 35 PM" src="https://user-images.githubusercontent.com/26833433/84352775-c9e3ea80-ab72-11ea-9224-51f3c05404e7.png">

It is very interesting that China represents such a large fraction of visits. It is a trend I noticed also in physics, with China investing heavily in numerous advanced research projects, including some worldwide SOTA examples: the Daya Bay and JUNO neutrino oscillation experiments, and the 500 meter FAST radio telescope. As a scientist I'm very happy to see this, as I believe knowledge discovered by any people will benefit all people.

**Comment by johntiger1** [comment]
好

**Comment by PuneetKohli** [comment]
The owners of this repo should seriously consider renaming it. Congratulations folks, you got the hype you wanted by misusing the YOLO name, now please change it to something more meaningful and appropriate. For what it's worth, your implementation seems to have runtime improvements, so call it something like `yolov4_pytorch_faster` or whatever else floats your boat.

**Comment by adamhrv** [comment]
yolo-ultra, or ultra-yolo
(It would help disambiguate)

**Comment by Zhangxiaof001** [comment]
很多人都在关注YOLO

**Comment by lucasjinreal** [comment]
I think yolov5 is OK, no matter how fancy model it use, but final AP tells the whole story.

**Comment by clw5180** [comment]
请问有哪位老哥测试过效果如何？

**Comment by theblackcat102** [comment]
@clw5180  you can refer to this comparison of YOLOv3, YOLOv4, "YOLOv5"
https://github.com/WongKinYiu/CrossStagePartialNetworks/issues/32#issuecomment-638064640

**Comment by WongKinYiu** [comment]
@theblackcat102 

The comparison is a little bit out-of-date. At the time, the comparison use May 27 YOLOv5. Now, YOLOv5 is updated to June 9 version.
![image](https://user-images.githubusercontent.com/12152972/84554288-bfc81480-ad49-11ea-8f4d-9f91cf91e79a.png)
It is better to see the comparison in https://github.com/ultralytics/yolov5/issues/6.

**Comment by huynhbach197** [comment]
![Uploading BB463FEC-5DF9-4489-8C4B-E830297DDFDA.jpeg…](

)

**Comment by huynhbach197** [comment]
![FA0F91DD-618A-4B4B-93EB-EC4DEC86B73A](https://user-images.githubusercontent.com/45510605/84592692-25b3b980-ae7a-11ea-8620-bcffc555fcf6.jpeg)

**Comment by glenn-jocher** [comment]
@huynhbach197 to be clear, **all 3 of these models are trained with ultralytics repositories**. Darknet yolov4 AP is much lower than these at 43.5 (which is not possible to see, since the poster strangely removed the y axis).

**Comment by glenn-jocher** [comment]
Independent analysis by Roboflow:
https://blog.roboflow.ai/yolov4-versus-yolov5/

**Comment by sunny0315** [comment]
Can the comparison of the following two pictures show that yolo5 has higher accuracy?(detectron2 faster-rcnn vs yolo5)
![image](https://user-images.githubusercontent.com/5812332/84638163-31aa8480-af29-11ea-88aa-f4329c0fe2c3.png)
![image](https://user-images.githubusercontent.com/5812332/84638189-3a9b5600-af29-11ea-85d1-c12dc86a6fd3.png)

**Comment by Franklin-Yao** [comment]
> what you said is awesome,but you can you up,no can no bb.

Dude, don't be rude.

**Comment by berserkr** [comment]
Yolo9000 is taken! 

> I think the project name should not be called yolov5
> Please add a link to yolov4：https://github.com/AlexeyAB/darknet

Call it YoloPlusUltra! Since Yolo9000 is taken :)

**Comment by chonyy** [comment]
Sorry but I still disagree with the idea of calling this repo Yolov5, it's just so confusing.

**Comment by dmvictor** [comment]
I took some time to figure out the relationship between YOLOv4 & v5. [This article](https://towardsdatascience.com/yolo-v4-or-yolo-v5-or-pp-yolo-dad8e40f7109) might be helpful for someone who has the same question.

**Comment by Curt-Park** [comment]
YOLOv5
YOLOv6
YOLOv7
...
Eventually,
https://github.com/juunini/YOLOv1972

**Comment by glenn-jocher** [comment]
@Curt-Park thanks for the suggestion! The YOLO naming convention has indeed gained widespread recognition. However, it's important to acknowledge the achievements of the YOLO community and the active contributions from each iteration. If you have any more feedback or questions related to YOLOv5, feel free to share!
